### ISSUE TITLE
Scancode ERROR: UNABLE TO IMPORT SCANCODE PLUGINS. in fossology 4.2.1.0 

### ISSUE THREAD

**Issue by dineshr93** [fossology/fossology#2348]
### Description
After installing the new version 4.2.1.0 of fossology we tested scan code for boost_1_64_0.tar.bz2 component. It fails with below error.

#### How to reproduce

For a bug: Describe the bug and list the steps you used when the issue occurred.

For an enhancement or new feature: Describe your needs/expected results.

#### Screenshots

![image](https://user-images.githubusercontent.com/4498415/204818789-f4cbe53a-29a9-4428-a75d-d562e36cc518.png)

### Versions

* Last commit id on master: Version: [4.2.1.0], Branch: [HEAD], Commit: [#9a0a6e] 2022/11/15 10:13 +00:00 built @ 2022/11/28 15:47
* Operating System (lsb_release -a): Ubuntu 18.04

### Logs

```
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "========================================================================"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "ERROR: UNABLE TO IMPORT SCANCODE PLUGINS."
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "Check your installation configuration (setup.py) or re-install/re-configure ScanCode."
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "The following plugin(s) are referenced and cannot be loaded/imported:"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "No module named 'gemfileparser'[0m"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "========================================================================"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "Traceback (most recent call last):"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "  File "/home/fossy/pythondeps/bin/scancode", line 5, in "
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "    from scancode.cli import scancode"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "  File "/home/fossy/pythondeps/scancode/cli.py", line 126, in "
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "    raise e"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "  File "/home/fossy/pythondeps/scancode/cli.py", line 113, in "
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "    plugin_classes, plugin_options = PluginManager.load_plugins()"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "  File "/home/fossy/pythondeps/plugincode/__init__.py", line 196, in load_plugins"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "    mgr_setup = manager.setup()"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "  File "/home/fossy/pythondeps/plugincode/__init__.py", line 222, in setup"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "    raise e"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "  File "/home/fossy/pythondeps/plugincode/__init__.py", line 220, in setup"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "    self.manager.load_setuptools_entrypoints(entrypoint)"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "  File "/home/fossy/pythondeps/pluggy/_manager.py", line 287, in load_setuptools_entrypoints"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "    plugin = ep.load()"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "  File "/home/fossy/pythondeps/importlib_metadata/__init__.py", line 208, in load"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "    module = import_module(match.group('module'))"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "  File "/usr/lib/python3.7/importlib/__init__.py", line 127, in import_module"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "    return _bootstrap._gcd_import(name[level:], package, level)"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "  File "/home/fossy/pythondeps/packagedcode/__init__.py", line 24, in "
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "    from packagedcode import cocoapods"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "  File "/home/fossy/pythondeps/packagedcode/cocoapods.py", line 21, in "
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "    from packagedcode import spec"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "  File "/home/fossy/pythondeps/packagedcode/spec.py", line 12, in "
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "    from gemfileparser import GemfileParser"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "ModuleNotFoundError: No module named 'gemfileparser'"
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: "FATAL scancode_wrapper.cc.94: could not execute scancode command: PYTHONPATH='/home/fossy/pythondeps/' SCANCODE_CACHE=/var/local/cache/fossology/scancode /home/fossy/pythondeps/bin/scancode -lc --custom-output - --custom-template scancode_template.html /srv/fossology/repository/localhost/files/b3/42/e2/b342e241ec605e3be3fa7aeb8eb38c254ffb514d.42e088cf38d5dae67d49dc21f1868220.6892 --quiet  --license-text --license-score 50 "
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: agent failed with error code 1
2022-11-30 11:30:25 scancode [0] :: JOB[1205].scancode[139405.localhost]: agent failed, code: 1
```

**Comment by shaheemazmalmmd** [comment]
@dineshr93 I hope you were able to solve the issue with scan code after the discussion ?

**Comment by dineshr93** [comment]
@shaheemazmalmmd  I think we haven't discussed on this issue. What steps need to be taken to resolve this  ?

**Comment by shaheemazmalmmd** [comment]
@dineshr93 Ah i just remembered We will look in to the issue.

**Comment by GMishx** [comment]
Should be fixed by #2436 and #2569
